src/org/parosproxy/paros/extension/ExtensionLoader.java

```diff
@@ -1263,6 +1266,14 @@ public void removeToolsMenuItem(JMenuItem menuItem) {
         View.getSingleton().getMainFrame().getMainMenuBar().getMenuTools().remove(menuItem);
     }
 
+    public void removeImportMenuItem(JMenuItem menuItem) {

```

Comment:
Not worth adding this method, there's no corresponding `add` also if an extension/add-on needs to dynamically add/remove an import menu then can do so by accessing the main menu bar.
(Yes, other remove methods should probably not exist either...)